### DISCUSSION TITLE
Welcome to fluidxml Discussions!

### DISCUSSION THREAD

**Discussion by daniele-orlando** [downforcetech/fluidxml.php#54] [Stories]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by kinekt4** [comment]
This discussion forum is a good idea. It allows users to find better better solutions before raising an issue.
